.github/workflows/tacos_unlock.yml

```diff
@@ -24,6 +24,17 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
+# FIXME: don't use the apply terraformer for the plan workflow
+# discussion: https://sentry.slack.com/archives/C05GNJCN9LM/p1705787896980429
```

Comment:
This is a public repo. This is not going to be particularly useful.
The description below is enough.

Reply:
I'll move it to gist.

Reply:
done